lib/ide-haskell-repl-view.coffee

```diff
@@ -37,7 +37,11 @@ class IdeHaskellReplView
     @editor.setGrammar \
       atom.grammars.grammarForScopeName 'source.haskell'
 
-    setTimeout (=> @editorElement.focus()), 100
+    @disposables.add atom.workspace.onDidChangeActivePaneItem (item) =>
+      if item == @
+        setTimeout =>
```

Comment:
[setImmediate](https://nodejs.org/api/timers.html#timers_setimmediate_callback_arg) is probably a better idea here.